custom_components/waste_collection_schedule/waste_collection_schedule/source/staedteservice_de.py

```diff
@@ -0,0 +1,116 @@
+import requests
+import datetime
+
+from waste_collection_schedule import Collection
+from waste_collection_schedule.service.ICS import ICS
+
+TITLE = "Städteservice"
+DESCRIPTION = "Städteservice Raunheim Rüsselsheim"
+URL = "https://www.staedteservice.de"
+
+TEST_CASES = {
+    "Rüsselsheim": {
+        "city": "Rüsselsheim",
+        "street_number": "411"
+    },
+    "Raunheim": {
+        "city": "Raunheim",
+        "street_number": "565"
+    },
+}
+
+BASE_URL = "https://www.staedteservice.de/abfallkalender"
+
+class Source:
+    def __init__(self, city, street_number):
+        self.city = str(city)
+        self.city_code = self.get_city_code()
+        self.street_number = str(street_number)
+        self.year = self.get_year()
```

Comment:
Why are you assigning year and month here? `__init__ `will be executed only once if Home Assistant starts.
I would do this all in fetch()